### ISSUE TITLE
Currently unable to sign into Google

### ISSUE THREAD

**Issue by JohanKool3** [softengsquad/studentmap#27]
Whenever I try to sign into google through the sign in button it throws this error.

![image](https://github.com/softengsquad/studentmap/assets/94987086/d936933f-6f04-4156-b1c9-5e50ff2d70b1)

**Comment by JohanKool3** [comment]
Yeah just ran it  in the <code>google_sign_in_fix</code> branch and it is still not working (same error).

**Comment by lewisclark** [comment]
> Yeah just ran it in the `google_sign_in_fix` branch and it is still not working (same error).

Ok. Could you send over your email address so we can try adding it to the whitelist?

**Comment by JohanKool3** [comment]
I am now able to sign in :).

**Comment by JohanKool3** [comment]
Just need to merge the branch to main that includes the changes.
